hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/SqlConnector.java

```diff
@@ -496,6 +497,34 @@ default List<String> getPrimaryKey(Table table) {
         throw new UnsupportedOperationException("PRIMARY KEY not supported by connector: " + typeName());
     }
 
+    /**
+     * @return
```

Comment:
add a Javadoc (including info that the set is immutable). `SqlConnector` is our internal SPI, someday it may become public API